### ISSUE TITLE
Fix/New project page numbers

### ISSUE THREAD

**Issue by lilblizzard** [KislakCenter/VisualCollation#73]
- Folio numbers are always generated when a new project, regardless of whether page number is selected instead
- Test what kind of request is being made on page number project init vs when the batch leaf request is made

**Comment by lilblizzard** [comment]
request with page number selected: `{"project":{"title":"test","shelfmark":"t123","notationStyle":"r-v","metadata":{"date":"1234"},"preferences":{"showTips":true,"side":{"page_number":true},"leaf":{"folio_number":true}}},"groups":[{"number":1,"leaves":10,"conjoin":true,"oddLeaf":1}],"folioNumber":1,"pageNumber":null,"startingTexture":"Hair"}`

**Comment by lilblizzard** [comment]
request with folio number selected: `{"project":{"title":"test","shelfmark":"t1234","notationStyle":"r-v","metadata":{"date":"1234"},"preferences":{"showTips":true,"side":{"page_number":true},"leaf":{"folio_number":true}}},"groups":[{"number":1,"leaves":10,"conjoin":true,"oddLeaf":1}],"folioNumber":1,"pageNumber":null,"startingTexture":"Hair"}`

**Comment by lilblizzard** [comment]
The problem is happening because neither of these requests are ever setting and sending a page number value, even when it is selected in the dialog. Check how ProjectOptions component assembles this data for the request, and amend it so it properly handles page number creation from the start.

**Comment by lilblizzard** [comment]
The radio button group for folio or page number generation:

`<RadioButtonGroup
          name="folioOrPage"
          defaultSelected="folio_number"
          onChange={(e, v) => handleChange(e, v)}
          style={{ width: 240 }}
        >
          <RadioButton
            aria-label="Generate folio numbers"
            value="folio_number"
            label="Generate folio numbers"
          />
          <RadioButton
            aria-label="Generate page numbers"
            value="page_number"
            label="Generate page numbers"
          />`

**Comment by lilblizzard** [comment]
The above handleChange function looks like this:
```
let handleChange = (e, v) => {              
    if (v === 'folio_number') {             
        props.set('generateFolioNumber', v);
    } else if (v === 'page_number') {       
        props.set('generatePageNumber', v); 
    } else {                                
        console.log(e, v);                  
    }                                       
};                                          
```

**Comment by lilblizzard** [comment]
closed by #80
